src/EA.Weee.Web/Controllers/OrganisationRegistrationController.cs

```diff
@@ -689,6 +689,8 @@ public async Task<ActionResult> ContactDetails()
             var countries = await GetCountries();
             model.AddressData.Countries = countries;
 
+            model.HasAuthorisedRepresentitive = existingTransaction?.AuthorisedRepresentative == YesNoType.Yes;
```

Comment:
@stevelee-civica  can you create a unit tests to ensure this value is set correctly...

Reply:
updated and added assertions on without existing transaction